app/handlers/profiles/settings/profbackground.go

```diff
@@ -74,7 +81,29 @@ func ProfileBackgroundSubmitHandler(c *gin.Context) {
 			slog.ErrorContext(c, err.Error())
 			return
 		}
-		saveProfileBackground(ctx, 1, fmt.Sprintf("%d.jpg?%d", ctx.User.ID, time.Now().Unix()))
```

Comment:
why was the unix timestamp attached to the end? is that important?

Reply:
ripple attempts to bypass caching pt 1 of 90

Reply:
kk we should test this but not a blocker/can do once the initial fix is out

Reply:
i think overall for `a.akatsuki.pw` we might want to send `cache-control: no-cache` headers